pkg/resource/graph/dependency_graph_test.go

```diff
@@ -97,7 +97,7 @@ func TestGraphNoDuplicates(t *testing.T) {
 
 	assert.Equal(t, []*resource.State{
 		b, c, d,
-	}, dg.DependingOn(a))
+	}, dg.DependingOn(a, nil))
```

Comment:
can we add some specific unit tests at the DG level of this specific parameter?

Reply:
Done.